Makefile

```diff
@@ -413,9 +413,12 @@ buildbox-grpc:
 	  --gofast_out=plugins=grpc:.\
     *.proto
 
-	cd lib/auth/proto && protoc -I=.:$$PROTO_INCLUDE \
-	  --gofast_out=plugins=grpc:.\
-    *.proto
+	protoc -I=.:$$PROTO_INCLUDE \
+		--proto_path=api/proto/auth \
+		--gofast_out=plugins=grpc:api/proto/auth \
+		--doc_out=api/docs \
+		--doc_opt=markdown,client.md \
```

Comment:
Pull this into a separate PR later, let's not conflate strict refactoring with new functionality.